wtxmgr/tx.go

```diff
@@ -768,7 +770,8 @@ func (s *Store) UnspentOutputs(ns walletdb.ReadBucket) ([]Credit, error) {
 		var rec TxRecord
 		err = readRawTxRecord(&op.Hash, recVal, &rec)
 		if err != nil {
-			return err
+			return fmt.Errorf("unable to retrieve transaction %v: "+
```

Comment:
distinguish the two by having this read `unable to read raw tx record: %v`?

Reply:
Good idea, done.